cpp/src/Ice/PluginManagerI.cpp

```diff
@@ -344,27 +305,7 @@ Ice::PluginManagerI::loadPlugin(const string& name, const string& pluginSpec, St
         cmdArgs = properties->parseCommandLineOptions(name, cmdArgs);
     }
 
-    PluginFactory factory = nullptr;
```

Comment:
The logic remains largely the same.

Before: factoryFunc comes from global map.
Now: factoryFunc is provided by initData